pkg/cli/client/image_cmd_test.go

```diff
@@ -113,6 +114,61 @@ func TestSignature(t *testing.T) {
 		So(err, ShouldBeNil)
 	})
 
+	Convey("Test with cosign signature(withReferrers)", t, func() {
+		currentWorkingDir, err := os.Getwd()
```

Comment:
Use a defer statement to return to currentWorkingDir. In case this test fails it will not affect other tests.
Please change everywhere in TestSignature